features/filiatie.feature

```diff
@@ -0,0 +1,470 @@
+
+        En Perceel met identificatie "P11" is gesplitst naar percelen "P12" en "P13"
+        En Perceel met identificatie "P12" is gesplitst naar appartementsrechten "A9" en "A10"
+        En Appartementsrecht met identificatie "A9" is niet verder gesplitst of verenigd
+        En Appartementsrecht met identificatie "A9" heeft "appartementsrechtVolgnummer"= 1
+        En Appartementsrecht met identificatie "A10" is niet verder gesplitst of verenigd
+        En Appartementsrecht met identificatie "A10" heeft "appartementsrechtVolgnummer"= 2
+
+        En Percelen met identificatie "P14" en "P15" zijn verenigd naar perceel "P16"
+        En Perceel met identificatie "P16" is gesplitst naar appartementsrechten "A11" en "A12"
+        En Appartementsrecht met identificatie "A11" is niet verder gesplitst of verenigd
+        En Appartementsrecht met identificatie "A11" heeft "appartementsrechtVolgnummer"= 1
+        En Appartementsrecht met identificatie "A12" is niet verder gesplitst of verenigd
+        En Appartementsrecht met identificatie "A12" heeft "appartementsrechtVolgnummer"= 2
+
+        En Perceel met identificatie "P17" is gesplitst naar appartementsrechten "A13", "A14" en "A15"
+        En Perceel met identificatie "P17" is niet ontstaan uit splitsing of vereniging van een ander perceel
+        En Appartementsrecht met identificatie "A13" is niet verder gesplitst of verenigd
+        En Appartementsrecht met identificatie "A14" is ondergesplitst in appartementsrechten "A16" en "A17"
+        En Appertementsrechten met identificatie "A15" en "A17" zijn samengevoegd (via wijzigingssplitsing)
+        En voor de wijzigingssplitsing zijn appartementsrechten "A13", "A14", "A15", "A16" en "A17" vervallen
+        En is appartementsrecht "A13" nieuw appartementsrecht "A18" geworden
+        En is appartementsrecht "A16" nieuw appartementsrecht "A19" geworden
+        En zijn appartementsrechten "A15" en "A17" nieuw appartementsrecht "A20" geworden
+        En Appartementsrecht met identificatie "A13" heeft "appartementsrechtVolgnummer"= 1
+        En Appartementsrecht met identificatie "A14" heeft "appartementsrechtVolgnummer"= 2
+        En Appartementsrecht met identificatie "A15" heeft "appartementsrechtVolgnummer"= 3
+        En Appartementsrecht met identificatie "A16" heeft "appartementsrechtVolgnummer"= 4
+        En Appartementsrecht met identificatie "A17" heeft "appartementsrechtVolgnummer"= 5
+        En Appartementsrecht met identificatie "A18" heeft "appartementsrechtVolgnummer"= 6
+        En Appartementsrecht met identificatie "A19" heeft "appartementsrechtVolgnummer"= 6
+        En Appartementsrecht met identificatie "A20" heeft "appartementsrechtVolgnummer"= 6
+        En Appartementsrecht met identificatie "A18" is niet verder gesplitst of verenigd
+
+
+    Rule: Na splitsing van een perceel wordt filiatie getoond bij het vervallen perceel en bij de nieuwe percelen
```

Comment:
Dit geldt toch niet alleen bij tonen?

Reply:
ik bedoel met "tonen" dat deze gegevens in het antwoord worden opgenomen

ik wijzig de rule naar "Rule: Na splitsing van een perceel wordt filiatie opgenomen bij het vervallen perceel en bij de nieuwe percelen". @MelvLee is dat beter?